src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,35 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bladeLength;
+    private int bladeWeight;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String name, String color, int bladeLength, int bladeWeight) {
+        setName(name);
+        setColor(color);
```

Comment:
more idiomatic approach would be using superclass' constructor
```suggestion
        super(name, color);
```
for that to work add constructors to Machine
```
    public Machine() {
    }

    public Machine(String name, String color) {
        this.name = name;
        this.color = color;
    }
```